### PR TITLE
fix(home): Fix issue where some browse cards are missing

### PR DESCRIPTION
**Pull request by dexter-mh-lee**
Added smoke tests to make sure this issue does not happen again. 

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: smoke-test/tests/cypress/example_to_datahub_rest.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+source:
+  type: "file"
+  config:
+    filename: "./data.json"
+
+# see https://datahubproject.io/docs/metadata-ingestion/sink_docs/datahub for complete documentation
+sink:
+  type: "datahub-rest"
+  config:
+    server: "http://localhost:8080"
```
Comment (gabe-lyons) [inline smoke-test/tests/cypress/example_to_datahub_rest.yml:11]: nice, thanks for adding this!

**Review by gabe-lyons — APPROVED** [review]

File: smoke-test/tests/cypress/example_to_datahub_rest.yml (hunk 1 above)
Comment (dexter-mh-lee) [reply smoke-test/tests/cypress/example_to_datahub_rest.yml:11]: hahaha

Comment (dexter-mh-lee) [reply smoke-test/tests/cypress/example_to_datahub_rest.yml:11]: this was a mistake, but should I leave it here?

Comment (jjoyce0510) [reply smoke-test/tests/cypress/example_to_datahub_rest.yml:11]: lol

**Pull request merged**